bootstrap/components/topology/storm-topology-component.json

```diff
@@ -55,6 +55,40 @@
             "defaultValue": "hdfs://localhost:9000/tmp/hbase"
           }
         ]
+      },
+      {
+        "uiName": "Clusters Security Config",
+        "fieldName": "clustersSecurityConfig",
+        "tooltip": "Clusters Security Configuration",
+        "isOptional": true,
+        "type": "array.object",
+        "defaultValue": null,
+        "fields": [
+          {
+            "uiName": "Cluster Name",
```

Comment:
I'm not sure. It might need UI side change.

@shahsank3t 
We would like to get hints for cluster names. 
If we can provide hints for `clustersSecurityConfig. clusterName` (please note that clustersSecurityConfig is an array) we can change the type of `clustersSecurityConfig. clusterName` to `enumstring` and put cluster names to the options.
Please let me know the possibility of implementing this.